plugins/modules/database/misc/redis_del.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2021, Andreas Botzner <andreas at botzner dot com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: redis_del
+short_description: Delete key from Redis
```

Comment:
I think it's better to add `state` to `redis_set`. Then you can use `state: present` with a value to set a key to a value, and use `state: absent` to delete a key.

Reply:
sounds good :+1: